zcash_primitives/src/note_encryption.rs

```diff
@@ -274,7 +294,7 @@ fn parse_note_plaintext_minus_memo(
     let v = (&plaintext[12..20]).read_u64::<LittleEndian>().ok()?;
 
     let mut rcm = FsRepr::default();
-    rcm.read_le(&plaintext[20..52]).ok()?;
+    rcm.read_le(&plaintext[20..COMPACT_NOTE_SIZE]).ok()?;
```

Comment:
20 = version + diversifier + value. Is it worth it to have a constant for that too?

Reply:
I considered doing that for both the `20` and the `12`, but it seemed like a lot of textual overhead for boundaries that weren't (for the moment) as important as the `CompactNote` boundary.